tests/test_study_parser.py

```diff
@@ -42,35 +45,23 @@ def test_load_manifest(manifest_path, raises):
 def test_manifest_data(manifest_key, raises):
     with mock.patch(
         "builtins.open", mock.mock_open(read_data=get_mock_toml(manifest_key))
-    ) as mock_file:
+    ):
         with raises:
             if manifest_key == "invalid_none":
                 parser = StudyManifestParser()
             else:
                 parser = StudyManifestParser("./path")
+
             expected = mock_manifests[manifest_key]
             assert parser.get_study_prefix() == expected["study_prefix"]
-            print(expected)
-            if "sql_config" in expected.keys():
-                if expected["sql_config"]["file_names"] == None:
```

Comment:
This big block got changed because pylint wanted this to be `is None` but while I was here, I thought I'd make these stanzas a little less branch-y.